OneCommunityGlobal/HGNRest
# Imran reset change password others

# Description
Gives the user permission to Reset and/or Change the password of any user but Owner/Admin classes. 

## Related PRS (if any):
To test this backend PR you need to checkout the frontend [PR#1808](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/pull/1808) .
…

## Main changes explained:
- added resetPassword permission to initialPermission and Test Permission
- Update resetPassword function to check for resetPassword permission 

## How to test:
1. check into current branch
2. do `npm install` and `...` to run this PR locally
3. follow the steps on the frontend [PR#1808](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/pull/1808) .
